### PR TITLE
Refactor of plugin

### PR DESCRIPTION
**Pull request by vsjolander**
- Removed support for varaibles alias.
- Using webworker

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,17 +7,20 @@
...
+  "author": {
+    "name": "Green Design System",
+    "email": "greendesignsystem@seb.se"
+  },
   "license": "MIT",
   "devDependencies": {
-    "@create-figma-plugin/build": "^2.6.1",
-    "@create-figma-plugin/tsconfig": "^2.6.1",
-    "@figma/plugin-typings": "^1.76.0",
+    "@create-figma-plugin/build": "^3.1.0",
```
Comment (alexanderczigler) [inline package.json:16]: I would suggest pinning your dependencies (and setting up Dependabot to manage updates). This gives you more control over dependencies' and subdependencies' versions.

File: src/types.ts [hunk 2]
```diff
@@ -1,155 +1,153 @@
-import {EventHandler} from '@create-figma-plugin/utilities'
-import {JSX} from "preact";
-import {InterimVariable} from "./classes/InterimVariable";
+import { EventHandler } from "@create-figma-plugin/utilities";
+import { JSX } from "preact";
+import { InterimVariable } from "./classes/InterimVariable";
```
Comment (astrit) [inline src/types.ts]: This is throwing an error: 

```shell
Cannot find module './classes/InterimVariable' or its corresponding type declarations.
3 import { InterimVariable } from "./classes/InterimVariable";
```

File: package.json (hunk 1 above)
Comment (vsjolander) [reply package.json:16]: I will add dependabot, not sure why it hasn't been added.

~~Regarding pinning: don't you miss out on secority and feature updates that aren't breaking?~~
I guess dependabot fixes this :)

File: src/types.ts (hunk 2 above)
Comment (vsjolander) [reply src/types.ts]: Forgot to save this file 🙈

File: package.json (hunk 1 above)
Comment (alexanderczigler) [reply package.json:16]: Well, in reality yes you might. In a perfect world package maintainers should update their dependencies accordingly and release patches... 🙃

**Pull request merged**